server_api.py

```diff
@@ -457,9 +457,9 @@ def register_no_right_user(data):
 
         db = mysql()
         db.connect()
-        sql = "SELECT count(*) FROM user WHERE user_name='" + str(data["bluetooth_id"]) + "'"
-        pure_result = db.query(sql)
-        if int(pure_result[0][0])<1:
+        with UserDao() as userDao:
+            pure_result = userDao.checkUserExisted(userName=str(data["bluetooth_id"]))
+        if int(pure_result)<1:
```

Comment:
"if not is_existed:" will be correct.
"if is_existed is None:" when is_existed == 0 it will be false.